src/index.js

```diff
@@ -5,14 +5,30 @@ const assign = require('object-assign');
 const findBabelConfig = require('find-babel-config'); // eslint-disable-line
 
 function getMappingFromBabel(start) {
+    // eslint-disable-next-line strict
+    'use strict';
+
     const c = findBabelConfig(start);
-    if (c && c.config && Array.isArray(c.config.plugins)) {
-        const pluginConfig = c.config.plugins.find(p => p[0] === 'module-alias');
-        // The src path inside babelrc are from the root so we have
-        // to change the working directory for the "current file directory"
-        // in order for the mapping in the editor (atom/sublime) to work properly
-        process.chdir(path.dirname(c.file));
-        return pluginConfig[1];
+    const env = process.env.BABEL_ENV || process.env.NODE_ENV || 'development';
+
+    if (c && c.config) {
+        let pluginConfig;
+
+        if (Array.isArray(c.config.plugins)) {
+            pluginConfig = c.config.plugins.find(p => p[0] === 'module-alias');
+        }
+
+        if (c.config.env && c.config.env[env] && Array.isArray(c.config.env[env].plugins)) {
+            pluginConfig = c.config.env[env].plugins.find(p => p[0] === 'module-alias');
```

Comment:
If babel finds a "default" configuration and a configuration specific to an environment, will it merge these 2 configurations? I believe so.

Would you mind adding a test checking that the right configuration is retrieved when having a specific environment config?